examples/integrations/itk/SelectROI.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    roi_slice
</pre><p>Let's add another cell for illustration that calls <code>viewer.get_roi_slice()</code> (no argument) that demonstrates we get the same result since that is the current scale. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>